modules/afsocket/socket-options-inet.c

```diff
@@ -159,6 +159,8 @@ socket_options_inet_setup_socket(SocketOptions *s, gint fd, GSockAddr *addr, AFS
       break;
     }
 #endif
+    default:
```

Comment:
IP address can be either ipv4 or ipv6. We should use g_assert_not_reached() here.